docs/ADR-48-locking-collections-in-the-builder.md

```diff
@@ -42,7 +42,29 @@ function isLocked(element: Collection | Item) {
 }
 ```
 
-Anyone consuming this data can read this property and disable all components that allow changes that would create a desync, like changing the name of a Decentraland Collection for example.
+### What's being locked
+
+As we said before, the locking mechanism is in place so we can avoid desynching the database with the blockchain. Anyone consuming this data can read this property and disable all components that change data that's being commited to the Blockchain.
+
+- **Decentraland**: Once the collection is published, every property that changes the address should not be updated. These are:
+
+  - The name
+  - Adding or removing items
+  - Changing items data:
+    - rarity
+    - price
+    - beneficiary
+    - metadata
+      - type
+      - name
+      - description
+      - data (the BodyShape and Category)
+
+- **Third Party**: Once the items are published, the URN cannot change anymore. That means that item `token_id`s and virtual collection `urn_suffix`es. For reference, and URN looks like this: `urn:decentraland:{network}:collections-thirdparty:{third-party-name}:{collection-id}:{item-id}` where
```

Comment:
I added it on each section, like this:

  - collection-id: the virtual collection's `urn_suffix`, *used as part of the full URN and locked after the first item is published as it will be used for all subsequent items*
  - item-id: the item's `token_id`, *editable up until publication*
  
you mean adding it here in the first paragraph too? (not against it, just asking)